neoload/commands/test_settings.py

```diff
@@ -161,3 +167,27 @@ def fill_default_fields(json_data):
         ('lgZoneIds', default_lgs(json_data.get('lgZoneIds'), json_data.get('controllerZoneId')))
     ])
     return data
+
+def createoruse(name, rename, description, scenario, controller_zone_id, lg_zone_ids, naming_pattern):
```

Comment:
The command createoruse does an update an not just "use". We should either
* call it "createorpatch"
* or remove the call to patch() and only set the id in the metadata

Reply:
Renamed to 'createorpatch' in [a9326b3]; great catch!